rpc/amqp-rpc.go

```diff
@@ -171,7 +173,7 @@ type AmqpRPCServer struct {
 
 // NewAmqpRPCServer creates a new RPC server for the given queue and will begin
 // consuming requests from the queue. To start the server you must call Start().
-func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer)) (*AmqpRPCServer, error) {
+func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer), durable, autoAck bool) (*AmqpRPCServer, error) {
```

Comment:
Why do we need to do these? Who will gain them?

Let's do this addition in another PR. It's enough of a change that I'd like to discuss it in detail.

(Tangentially, blind booleans like this are very confusing. We'll want to come up with config struct and maybe even "fluent" setters for it.)